package/contents/ui/main.qml

```diff
@@ -350,7 +359,11 @@ Item {
     }
 
     function togglePinToAllDesktops() {
-        tasksModel.requestVirtualDesktop(tasksModel.activeTask, 0);
```

Comment:
ok, I have merged in master a way to know the plasma desktop version in which the applet is running. To keep compatibility with older plasma version, this can be changed to:

```
if (root.plasma515) {
     tasksModel.requestVirtualDesktops(tasksModel.activeTask, 0);
} else {
     tasksModel.requestVirtualDesktop(tasksModel.activeTask, 0);
}
```

update the PR please and afterwards I think we can merge